firstvoices/backend/models/sites.py

```diff
@@ -132,6 +133,14 @@ class Meta:
     # from fvdialect:contact_email
     contact_email = models.EmailField(null=True, blank=True)
 
+    homepage = models.OneToOneField(
+        SiteWidgetList,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="site_homepage",
```

Comment:
These reverse names are always hard. Maybe this should be something like "homepage_of" or "homepage_site" ?

Reply:
Good call. I will update this.